inst/pages/Introductory-workflow.qmd

```diff
@@ -0,0 +1,353 @@
+
+```{r}
+#| label: subsetBySample
+tse_subset_by_sample <- tse[ , tse$cohort =="Cohort_1"]
+```
+
+This will create a *TreeSummarizedExperiment* Object only containing samples of the first cohort.
+
+## Agglomerating data
+
+To further push your data analysis and focus on its distribution at a specific taxonomic rank, it can be beneficial to agglomerate your data to that particular level. The agglomerateByRank function simplifies this process, enabling more streamlined and effective analyses. Here’s an example:
+
+```{r}
+#| label: agglomerating-data
+tse.agglomerated <- agglomerateByRank(tse, rank='Phylum')
+
+# Check
+datatable(data.frame(rowData(tse.agglomerated)),options = list(pageLength = 5),rownames = FALSE)
+```
+
+Great! Now, our data is confined to the taxonomic information up to the Phylum level, allowing the analysis to be focused on this specific rank. In the rest of the workflow, we won't be using the agglomerated data but all of the code beneath can be used on this.
+
+# Metrics
+
+## Community Diversity
+
+Community diversity in microbiology is measured by several indices:
+
+-   species richness (total number of species)
+-   equitability (distribution of species within a microbiome)
+-   diversity (combination of the two)
+
+The **coefficient of @Hill** combines these measures into a single equation. All these variations are referred to as alpha diversity.
+
+```{r}
+#| label: calculateRichness
+tse_alpha <- Tengeler2020
+
+# Estimate (observed) richness
+tse_alpha <- mia::estimateRichness(tse_alpha, 
+                             assay.type = "counts", 
+                             index = "observed", 
+                             name="observed")
+
+# Check some of the first values in colData
+head(tse_alpha$observed)
+```
+
+The result is showing the estimated richness values for different samples or locations within the dataset. It gives an idea of how diverse each sample is in terms of the number of different species present. We can then create a graph to visualize this.
+
+```{r}
+#| label: loadScater
+#| output: false
+library(scater)
+```
+
+```{r}
+#| label: plotColdata
+plotColData(tse_alpha, 
+            "observed", 
+            "cohort", 
+            colour_by = "patient_status") +
+  theme(axis.text.x = element_text(angle=45,hjust=1)) + 
+  labs(y=expression(Richness[Observed]))
+```
+
+To go even further, we can also compare the estimated Shannon index to the observed richness . Shannon quantifies the diversity in terms of both the number of different species (richness) and the evenness of their distribution (abundance) and is calculated as such:
+
+$$
+H' = -\sum_{i=1}^{R} p_i \ln(p_i)
+$$ p~i~ being the proportion of a certain microorganism.
+
+<hr/>
+
+First, we can easily calculate this measure and add them to our TreeSE.
+
+```{r}
+#| label: calculateDiversity
+tse_alpha <- mia::estimateDiversity(tse_alpha, 
```

Comment:
Sometimes you use package::function and sometimes not. Might be better to not use